apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -128,8 +130,8 @@ function overrideRevisions( calypsoPort ) {
 
 		if ( action === 'loadRevision' && payload ) {
 			const blocks = parse( payload.content );
-			dispatch( 'core/editor' ).editPost( payload );
-			dispatch( 'core/editor' ).resetBlocks( blocks );
+			dispatch( editorStore ).editPost( payload );
+			dispatch( blockEditorStore ).resetBlocks( blocks );
```

Comment:
Did you mean `editorStore`?

```suggestion
			dispatch( editorStore ).resetBlocks( blocks );
```